beacon-chain/rpc/eth/shared/structs.go

```diff
@@ -65,6 +65,14 @@ type SyncCommitteeSubscription struct {
 	UntilEpoch           string   `json:"until_epoch" validate:"required,number,gte=0"`
 }
 
+type BeaconCommitteeSubscription struct {
+	ValidatorIndex   string `json:"validator_index" validate:"required,number,gte=0"`
+	CommitteeIndex   string `json:"committee_index" validate:"required,number,gte=0"`
+	CommitteesAtSlot string `json:"committees_at_slot" validate:"required,number,gte=0"`
+	Slot             string `json:"slot" validate:"required,number,gte=0"`
+	IsAggregator     bool   `json:"is_aggregator"`
```

Comment:
no `validate: "required"` here because adding it to a bool requires the field's value to be `true`